lambdas/seed/shared/helpers/ResponseHelper.js

```diff
@@ -0,0 +1,52 @@
+const DEFAULT_HEADERS = {
+  'Access-Control-Allow-Origin': '*',
+  'Access-Control-Allow-Headers': '*',
+  'Content-Type': 'application/json',
+};
+
+/**
+ * The ResponseHelper contains methods for properly returning data from the
+ * HTTP request.
+ */
+class ResponseHelper {
+  /**
+   * Constructs the response object that each Lambda function needs to return.
+   *
+   * @param {Error} err The error object or string with message in case there was an error.
+   * @param {res} res The response body that shall be returned in case of success.
+   */
+  static createResponse(err, res) {
+    return {
+      statusCode: err ? '400' : '200',
```

Comment:
bug: we should be passing in a statusCode (or error from the Errors file) instead of defaulting to a `400`. There may be instances we call this function and its an internal error but the user sees a `400` instead of a `500` response.

Reply:
That's correct! It will be fixed. The same applies to `createErrorResponse()`